gradle.properties

```
@@ -2,7 +2,7 @@
 # -> https://plugins.jetbrains.com/docs/intellij/intellij-artifacts.html
 pluginGroup=com.dprint.intellij.plugin
 pluginName=dprint
-pluginVersion=0.8.2
+pluginVersion=0.8.3
```

Super nit: I tend to roll a beta version until I am ready to release

```suggestion
pluginVersion=0.8.3.beta
```